in_text/text_data_release.yml

```
@@ -36,208 +36,464 @@ file-format: Four zipped folders and associated files.
+        This release
+      data-min: -0.903492321
+      data-max: -0.140221919
+      data-units: XX
+    -
+      attr-label: source
+      attr-def: >-
+        XX
+      attr-defs: >-
+        This release
       data-min: NA
       data-max: NA
       data-units: NA
     -
-      attr-label: hyperparameter_config.yaml
+      attr-label: sink
       attr-def: >-
-        A configuration file for hyperparameter optimization.
+        XX
       attr-defs: >-
-        citation or URL of z. If citation, should appear in cross-cites. Can use NA or This release
+        This release
       data-min: NA
       data-max: NA
       data-units: NA
     -
-      attr-label: model_config.yaml
+      attr-label: year
       attr-def: >-
-        A configuration file for running machine learning model.
+        XX
       attr-defs: >-
-        citation or URL of z. If citation, should appear in cross-cites. Can use NA or This release
-      data-min: NA
-      data-max: NA
+        This release
+      data-min: 2001
+      data-max: 2020
       data-units: NA
-  -
-    data-name: 04_analysis.zip
-    data-description: >-
-      A zip file containing python scripts and configuration files for analyzing model performance including measuring functional performance and calculating expected gradients.
-    attributes: 
     -
-      attr-label: src
+      attr-label: lag
       attr-def: >-
-        A folder containing python scripts to run information theory analysis and other analyses.
+        XX
       attr-defs: >-
-        citation or URL of y. If citation, should appear in cross-cites. Can use NA or This release
-      data-min: NA
-      data-max: NA
+        This release
+      data-min: 0
+      data-max: 9
       data-units: NA
-  -
-    data-name: README.md
-    data-description: >-
-      A markdown file detailing how to run the included workflow.
-    attributes: 
-  -
-    data-name: Snakefile_fetch_munge
-    data-description: >-
-      A snakemake workflow used to fetch and process data for machine learning model.
-    attributes: 
-  -
-    data-name: Snakefile_model_analysis
-    data-description: >-
-      A snakemake workflow used to analyze model performance using information theory analysis, expected gradients, and other techniques.
-    attributes:
-  -
-    data-name: Snakefile_run_ml_model
-    data-description: >-
-      A snakemake workflow used to run machine learning model.
-    attributes:
-  -
-    data-name: environment.yaml
-    data-description: >-
-      An environment yaml for creating a conda environment with the necessary libraries.
-    attributes:
-  -
-    data-name: utils.py
-    data-description: >-
-      A python script with helper functions for retrieving and processing data.
-    attributes:
 
 purpose: This release provides code and data for a running a machine learning model to predict estuary salinity and using information theory, explainable AI, and comparison to other modeling approaches for analysis. 
```

Removed the "a" before "machine learning model".
```suggestion
purpose: This release provides code and data for running a machine learning model to predict estuary salinity and using information theory, explainable AI, and comparison to other modeling approaches for analysis. 
```